mw-webapp/src/component/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from "src/component/checkbox/Checkbox.module.scss";
+
+/**
+ * Checkbox props
+ */
+interface CheckboxProps {
+  /**
+   * Checkbox status
+   */
+  checked: boolean;
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Checkbox`s value
+   */
+  value?: string;
+  /**
+   * Checkbox's name
+   */
+  name?: string;
+  /**
+   * The flag sets the element to be required
+   */
+  required?: boolean;
```

Comment:
what will happened if it is required?
what will happened if it is not required?

Reply:
I just asked how does it work. Please, share you vision. But before sharing think about it and you will find weak spots - change them

Reply:
I see the use of this property in forms when we need to block form submission without selecting a required field with a checkbox.

Reply:
Removed property

Reply:
we are going to use it everywhere  not in forms only